Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -15,6 +14,12 @@ let DefaultTimeoutTimeInterval = 10.0 // Seconds.  We'll want some telemetry on
  * the user to navigate back to Settings.
  */
 class SettingsContentViewController: UIViewController, WKNavigationDelegate, Themeable {
+    private struct UX {
+        static let errorLeadingAnchor: CGFloat = 20
```

Comment:
It's not an anchor, it's a constant and I think it's better to declare just one constant e.g. errorLeadingTrailingPadding: CGFloat = 20.